src/main/java/keycontacts/logic/commands/FindCommand.java

```diff
@@ -18,8 +19,8 @@ public class FindCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all students whose names contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "Parameters: Name, Address, Phone Number, Grade Level...\n"
+            + "Example: " + COMMAND_WORD + " [prefix/keyword]";
```

Comment:
Should probably match the other commands' `MESSAGE_USAGE`

Reply:
Could you give it a check